### DISCUSSION TITLE
Sonoff Dual R2 - Chain Actions

### DISCUSSION THREAD

**Discussion by pksml85** [arendst/Tasmota#14249] [Q&A] [answered]
Hello fellow Sonoff users. I have a motorized projector screen. It used to be controlled with a toggle switch that would send mains voltage on one wire or the other. The Sonoff Dual R2 has taken over this responsibility w/ the help of Tasmota. It's awesome!
(BTW, I'm using the stock tasmota.bin file v10.0.0)

Here's my concern and question:

If both relays get turned on at the same time, it could be disastrous and potentially dangerous. To help mitigate this, I've set the PulseTime of both relays to 15 seconds, which gives enough time for the motor to bring the screen all the way up/down.

Note: The one button on the device will activate one of the relays with a single press or the other with a double press.

Is there a way in Tasmota to turn relay 2 off (or confirm it's off) when I activate relay 1 and vice versa?

**Comment by rapi3** [comment, marked as answer]
yes it is; have a look in documentation at **Interlock**

**Reply by pksml85** [reply]
That's an interesting idea. Didn't know about that feature (the commands page is quite daunting :)

I'm guessing interlock is a way of grouping relays together. So turning one on will also turn on another/others???

(Note that I need to make sure when one is turned on that the other is *not* on.)

Edit: OK, I found [this link](https://github.com/arendst/Tasmota/issues/6313#issuecomment-525946076) and [this link](https://help.ewelink.cc/hc/en-us/articles/360041366612-Inching-and-Interlock-mode) which confirms that interlock is exactly the setting I need.
(The Tasmota commands page doesn't explain what interlock mode is exactly...)

Thanks @rapi3!
